conf/test_copro.config

```diff
@@ -0,0 +1,44 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/nascent -profile test,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'CoPRO PINTS Test profile'
+    config_profile_description = 'Test dataset to check PINTS pipeline function(https://pints.yulab.org/tre_calling#part-iv-case-2)'
+
+    // Input data
+    input = "${projectDir}/tests/samplesheets/copro.csv"
+
+    genome = 'hg38'
+    assay_type = 'CoPRO'
+    filter_bed = "https://pints.yulab.org/ref/examples/promoters_1kb_tss_centered.bed.gz"
+    with_umi = true
+    umitools_dedup_stats = true
+}
+
+process {
+    withName: FASTP {
```

Comment:
A small trick here. Use the full process name in test profiles, to make sure it takes priority over configuration loaded in the `modules.config` when using the simple name there.